pkg/chartrenderer/default.go

```diff
@@ -138,8 +136,10 @@ func (r *chartRenderer) renderResources(ch *chartapi.Chart, values chartutil.Val
 		}
 	}
 
-	manifests := manifest.SplitManifests(files)
-	manifests = SortByKind(manifests)
+	_, manifests, err := releaseutil.SortManifests(files, chartutil.DefaultVersionSet, releaseutil.InstallOrder)
+	if err != nil {
+		return nil, err
```

Comment:
It is nice to wrap the error with a more meaningful message what failed.

```suggestion
		return nil, fmt.Errorf("failed to sort manifests: %w, err)
```